src/app/components/card-item/card-item.component.scss

```diff
@@ -0,0 +1,31 @@
+@import "src/styles/_variables.scss";
+@import "src/styles/_colors.scss";
+@import "src/styles/_reuse.scss";
+
+.date {
+  position: relative;
+  font-size: 0.875rem;
+  line-height: 1rem;
+  color: $primary-gray;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  gap: 2.125rem;
+  width: 80%;
+  margin: auto;
+  padding-top: 2.125rem;
+
+  > div {
```

Comment:
Please use class instead of adding style properties directly to any tags.

Reply:
done